server/config/forms/incidentDetailsForm.js

```diff
@@ -41,6 +41,7 @@ const requiredIncidentDate = joi
             [ValidationError.isNotNumber]: 'Enter hours using numbers only',
             [ValidationError.isTooLarge]: 'Enter an hour which is 23 or less',
             [ValidationError.isNot2Digits]: 'Enter the hours using 2 digits',
+            [ValidationError.isNotPositiveNumber]: 'Enter the hours using positive values only',
```

Comment:
`Enter an hour which is 0 or more` might be more in keeping with the tooLarge message?

did you check with content about this?

Reply:
I didn't but I don't think there is a need to. I think the change is small enough to implement without them

Reply:
I didn't but I don't think there is a need to. I think the change is small enough to implement without them